tracing-fmt/Cargo.toml

```diff
@@ -4,6 +4,19 @@ version = "0.0.1-alpha.1"
 authors = ["Eliza Weisman <eliza@buoyant.io>"]
 edition = "2018"
 license = "MIT"
+documentation = "https://docs.rs/tracing-fmt/0.0.1-alpha.1/tracing-fmt"
+description = """
+An experimental subscriber for tracing, a runtime diagnostics system.
+"""
+categories = [
+    "development-tools::debugging",
+    "development-tools::profiling",
```

Comment:
should probably remove the "profiling" category from the `fmt` crate.

Reply:
done